Tests/DatadogTests/Datadog/Core/Upload/DataUploadStatusTests.swift

```diff
@@ -122,33 +122,32 @@ class DataUploadStatusTests: XCTestCase {
         try alertingStatusCodes.forEach { statusCode in
             let requestID: String = .mockRandom()
             let status = DataUploadStatus(httpResponse: .mockResponseWith(statusCode: statusCode), ddRequestID: requestID)
-            let error = try XCTUnwrap(status.internalMonitoringError, "Internal Monitoring error should be created for status code \(statusCode)")
-            XCTAssertEqual(error.message, "Data upload finished with status code: \(statusCode)")
-            XCTAssertEqual(error.attributes?["dd_request_id"], requestID)
+            let error = try XCTUnwrap(status.telemetryError, "Internal Monitoring error should be created for status code \(statusCode)")
```

Comment:
Let's rename `*Internal Monitoring*` → `*Telemetry*` in assertions and in test names (needs multiple edits in this file).

Reply:
ah good catch!